apps/web/src/components/nav/VisibilityButton.tsx

```diff
@@ -0,0 +1,31 @@
+import { IconVisibilityOff } from '@novu/design-system';
+import { css } from '../../styled-system/css';
+
+const buttonStyles = css({
+  bg: 'transparent',
+  transition: 'background 0.2s ease',
+  _hover: {
+    '& svg': {
+      fill: 'typography.text.secondary',
+    },
+  },
+  borderRadius: '50',
```

Comment:
is it meant to be `50%` ?

Reply:
It is not -- "50" is one of the `size` values we have in the design-system, but in this case, it doesn't seem to be getting picked up by borderRadius